mass_mailing_sending_queue/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Mass mailing sending queue
+==========================
+
+This module adds a sending queue when mass mailing 'Send to All' button is
+clicked. This is an aditional queue, there is another one for sending emails.
```

Comment:
`This is an additional queue, apart from the existing one for doing the actual sending.`